docs/testing.rst

```diff
@@ -381,6 +381,66 @@ functionality, the AioHTTPTestCase is provided::
    Handles executing an asynchronous function, using
    the :attr:`AioHTTPTestCase.loop` of the :class:`AioHTTPTestCase`.
 
+Patching unittest test cases
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Patching test cases is tricky, when using python older than 3.8 ``'patch'`` does not behave as it has to.
+We recommend using asynctest_ that provides a patch method that is capable of creating
+a magic mock that is async capable. It can be used with decorator as well as with context manager.::
```

Comment:
Having a period followed by a colon `.:` looks weird. did you mean
```suggestion
a magic mock that supports async. It can be used with decorator as well as with context manager::
```